cmd/root_test.go

```diff
@@ -23,3 +23,17 @@ func TestConfig(t *testing.T) {
 	_, ok = ethType.(*ethereum.ChainConfig)
 	require.True(t, ok)
 }
+
+func TestBlockQueueChannelSize(t *testing.T) {
+	file, err := cmd.Parse("../config/sample-config.yaml")
+	require.NoError(t, err, "Error parsing config")
+
+	var nobleCfg interface{} = file.Chains["noble"]
```

Comment:
with the introduction of generics in Go you can now use the `any` keyword, which is an alias for `interface{}`. totally a preference thing and you're likely to see both used everywhere but figured i'd point this out since `any` is shorter and has a slightly clearer meaning to the reader  vs. `interface{}`

Reply:
Nice! Appreciate the insight!
Using `any`.